VRDR/DeathRecord.cs

```diff
@@ -7388,6 +7758,1877 @@ public string EmergingIssue20
+  [PropertyParam("code", "The code used to describe this concept.")]
+  [FHIRPath("Bundle.entry.resource.where($this is Observation).where(code.coding.code='codedraceandethnicity')", "")]
+  public string HispanicCodeForLiteralHelper
+  {
+      get
+      {
+          if (HispanicCodeForLiteral.ContainsKey("code"))
+          {
+              return HispanicCodeForLiteral["code"];
+          }
+          return null;
+      }
+      set
+      {
+          SetCodeValue("HispanicCodeForLiteral", value, VRDR.ValueSets.HispanicOrigin.Codes);
+      }
+  }
+
+
+
+
+
+        /// <summary>Entity Axis Cause Of Death</summary>
+        /// <value>Entity-axis codes</value>
+        /// <example>
+        /// <para>// Setter:</para>
+        /// <para>Tuple<string, string, string, string>[] eac = new Tuple<string, string, string, string>{Tuple.Create("value", "position", "line", "code")}</para>
+        /// <para>ExampleDeathRecord.EntityAxisCauseOfDeath = eac;</para>
+        /// <para>// Getter:</para>
+        /// <para>Console.WriteLine($"Emerging Issue Value: {ExampleDeathRecord.EntityAxisCauseOfDeath}");</para>
+        /// </example>
+        [Property("Entity Axis Cause of Death", Property.Types.Tuple4Arr, "Entity Axis Cause of Death", "", true, IGURL.EntityAxisCauseOfDeath, false, 50)]
+        [FHIRPath("Bundle.entry.resource.where($this is Observation).where(code.coding.code=80356-9)", "")]
+        public Tuple<string, string, string, string>[] EntityAxisCauseOfDeath
+        {
+            get
+            {
+                List<Tuple<string, string, string, string>> eac = new List<Tuple<string, string, string, string>>();
+                if (EntityAxisCauseOfDeathObs != null)
+                {
+                    foreach(Observation ob in EntityAxisCauseOfDeathObs)
+                    {
+                        CodeableConcept valueCC = (CodeableConcept)ob.Value;
+                        string value = valueCC.Coding[0].Code;
+                        Observation.ComponentComponent positionComp = ob.Component.Where(c => c.Code.Coding[0].Code=="position").FirstOrDefault();
+                        string position = positionComp.Value.ToString();
+                        Observation.ComponentComponent lineNumComp = ob.Component.Where(c => c.Code.Coding[0].Code=="lineNumber").FirstOrDefault();
+                        string lineNumber = lineNumComp.Value.ToString();
+                        Observation.ComponentComponent codeComp = ob.Component.Where(c => c.Code.Coding[0].Code=="eCodeIndicator").FirstOrDefault();
+                        string code = codeComp.Value.ToString();
+                        Tuple<string, string, string, string> eacod = Tuple.Create(lineNumber, position, value, code);
+                        eac.Add(eacod);
+                    }
+                }
+                return eac.ToArray();
```

Comment:
What happens if there are more than the 20 allowed?

Reply:
They would get truncated in IJEMortality, do we want to add a validation error for something like that?